src/Chainweb/Version.hs

```diff
@@ -512,7 +512,7 @@ chainwebGraphs (TimedCPM g) = pure (0, g)
 chainwebGraphs (FastTimedCPM g) = pure (0, g)
 chainwebGraphs Testnet04 = pure (0, petersonChainGraph)
 chainwebGraphs Mainnet01 = pure (0, petersonChainGraph)
-chainwebGraphs Development = (50, twentyChainGraph) NE.:| [ (0, petersonChainGraph) ]
+chainwebGraphs Development = (twentyChainUpgradeBlockHeight Development, twentyChainGraph) NE.:| [ (0, petersonChainGraph) ]
```

Comment:
Isn't this the wrong way around? I think, the upgrades should be specified here and `twentyChainUpgradeBlockHeight` should be derived from it.